lib/build.js

```diff
@@ -8,7 +8,16 @@ const pify = require('pify');
 const writeFile = pify(fs.writeFile);
 
 module.exports = context => {
-	const dockerfile = `FROM node:${context.version}-onbuild`;
+	const dockerfile = [
+		`FROM node:${context.version}`,
+		'RUN mkdir -p /usr/src/app',
+		'WORKDIR /usr/src/app',
+		'COPY package.json /usr/src/app/',
+		'RUN npm install && npm cache clean --force',
+		'COPY . /usr/src/app',
```

Comment:
Same as https://github.com/vadimdemedes/trevor/pull/68/files#r200885884, can be `COPY . .`

Reply:
Updated.